plugins/search/src/components/SearchBar/SearchBar.test.tsx

```diff
@@ -210,4 +217,128 @@ describe('SearchBar', () => {
       expect.objectContaining({ term: value }),
     );
   });
+
+  it('does not capture google analytics event if not enabled in app', async () => {
```

Comment:
Hmm...  Is this test relevant still?

Reply:
I should be able to trust the analytics api, so theoretically I could delete it, even though I guess it doesn't hurt making sure we don't track events if not configured?

Reply:
🤷‍♂️  Yeah, either way. No harm!